localstack-core/localstack/services/cloudformation/engine/v2/change_set_model_preproc.py

```diff
@@ -223,7 +249,40 @@ def _resolve_condition(self, logical_id: str) -> PreprocEntityDelta:
             return condition_delta
         raise RuntimeError(f"No condition '{logical_id}' was found.")
 
+    def _resolve_pseudo_parameter(self, pseudo_parameter_name: str) -> PreprocEntityDelta:
+        # TODO: review required behaviour of before and after, are pseudo parameter values constant during
+        #       change set lifecycles?
+        match pseudo_parameter_name:
+            case "AWS::Partition":
+                after = "aws"
```

Comment:
What about our other partition support like govcloud? Maybe leave a todo. What does the v1 engine do here?

Reply:
Thanks, indeed we can use the `get_partition` utility about the region name as the v1 provider does